src/taxonomy/TaxonomyListPage.jsx

```diff
@@ -1,19 +1,69 @@
 import React, { useContext } from 'react';
 import {
+  Button,
   CardView,
   Container,
   DataTable,
+  Dropdown,
+  OverlayTrigger,
   Spinner,
+  Tooltip,
 } from '@edx/paragon';
+import {
+  Add,
+} from '@edx/paragon/icons';
 import { useIntl } from '@edx/frontend-platform/i18n';
 import { Helmet } from 'react-helmet';
 import SubHeader from '../generic/sub-header/SubHeader';
 import getPageHeadTitle from '../generic/utils';
 import messages from './messages';
 import TaxonomyCard from './taxonomy-card';
+import { getTaxonomyTemplateApiUrl } from './data/api';
 import { useTaxonomyListDataResponse, useIsTaxonomyListDataLoaded, useDeleteTaxonomy } from './data/apiHooks';
 import { TaxonomyContext } from './common/context';
 
+const TaxonomyListHeaderButtons = () => {
+  const intl = useIntl();
+  return (
+    <>
+      <OverlayTrigger
+        placement="top"
+        overlay={(
+          <Tooltip>
+            {intl.formatMessage(messages.downloadTemplateButtonHint)}
+          </Tooltip>
+        )}
+      >
+        <Dropdown>
+          <Dropdown.Toggle
+            variant="outline-primary"
+            data-testid="taxonomy-download-template"
+          >
+            {intl.formatMessage(messages.downloadTemplateButtonLabel)}
+          </Dropdown.Toggle>
+          <Dropdown.Menu>
+            <Dropdown.Item
+              href={getTaxonomyTemplateApiUrl('csv')}
+              data-testid="taxonomy-download-template-csv"
+            >
+              {intl.formatMessage(messages.downloadTemplateButtonCSVLabel)}
+            </Dropdown.Item>
+            <Dropdown.Item
+              href={getTaxonomyTemplateApiUrl('json')}
+              data-testid="taxonomy-download-template-json"
+            >
+              {intl.formatMessage(messages.downloadTemplateButtonJSONLabel)}
+            </Dropdown.Item>
```

Comment:
Adding `download=template.json/.csv`  or just download will also enforce that the browser will downlod the file instead of opening it in a tab.

Reply:
Thank you for the review @xitij2000 !

We are doing it in the backend here: https://github.com/openedx/openedx-learning/blob/3e2d6a497efddfae7a1c24b361ddfd36704dbb80/openedx_tagging/core/tagging/rest_api/v1/views_import.py#L47 using `Content-Disposition`. Should we also do it here?